### PR TITLE
Merge Rafa's changes

### PR DESCRIPTION
**Pull request by dcdenu4**
Okay, I think I got Rafa's changes to PR into release/3.12.0. Some changes did not make it because they were changes to sections that had been removed. Attached is what the current SDR html looks like. Some things to note in this PR that would be helpful to double check!

- Some equation references were failing, so I took a best stab at replacing them with the correct ones. PLEASE check those changes!
- Rafa, sorry I couldn't capture all your changes because of the above mentioned removed sections. Please read through and make sure you're happy with everything that was captured!

Under `Files Changed` tab you can view the raw changes and if clicking the page icon to right by the `<>` icon, you can view rich marked up changes. Note these marked up changes are doing their best to render but won't exactly reflect our html result. BUT, it's useful for better seeing text changes.

### CODE REVIEW

**Review by newtpatrol — APPROVED** [review]
Thanks @schmittrjp and @dcdenu4! It is definitely hard to tell what the final version will look like, but I'm happy to have these merged - it looks like Rafa did a lot of useful wordsmithing.

**Comment by dcdenu4** [comment]
Thanks @newtpatrol , I'll let @phargogh take a quick look at the equation references I swapped out before merging through.

File: source/sdr.rst
Comment (phargogh) [inline source/sdr.rst:241]: The `\right` is unbalanced here but also no longer needed now that `\left` has been removed.

```suggestion
.. math:: T_i=dT_i\cdot\left(\sum_{j\in\{pixels\ that\ drain\ to\ i\}}F_j \cdot p(i,j)))

```

File: source/sdr.rst
Comment (phargogh) [inline source/sdr.rst:252]:
```suggestion
Figure 4. Illustration of relevant sediment erosion and deposition processes, their spatial interconnections, and their representation in the model. The maximum amount of sediment that could be eroded from a pixel is defined as USLE value in the absence of vegetation (RKLS), the difference between that and actual erosion with landcover and management (RKLSCP) indicates the role of those local factors to avoid erosion. Of the sediment leaving a pixel (RKLSCP), only a fraction (SDR) reaches a downslope stream pixel. The remainder (:math:`RKLSCP*(1-SDR)`) is retained on downstream pixels. Thus, the role of vegetation is two-fold: (1) avoiding local erosion and (2) trapping sediment that was mobilized upslope. The box at the bottom indicates the potential fate of eroded sediment 

```

File: source/sdr.rst
Comment (phargogh) [inline source/sdr.rst:488]: All the bare soil stuff has been removed from the model, so we can remove these lines.  The bare soil stuff was only used in those "legacy outputs".

File: source/sdr.rst
Comment (phargogh) [inline source/sdr.rst:501]: Same here, this output has been removed.

File: source/sdr.rst
Comment (phargogh) [inline source/sdr.rst:515]: And this one too.

**Review by phargogh — CHANGES_REQUESTED** [review]
Thanks @dcdenu4 !  The equations look good to me, although we're also removing the bare soil outputs in https://github.com/natcap/invest/pull/1038 so I think we should remove them here as well.  And that resolves some of the equation reference questions too.

I also had a couple minor suggestions with the math to improve formatting.  Looks good otherwise!

**Comment by dcdenu4** [comment]
Thanks @phargogh , I think I got everything you mentioned. I also had to do some merge-smithing from recent changes @newtpatrol made to `natcap/release/3.12.0`. But if it all looks mostly okay, we should probably get this PR'd in so folks can continue to tinker with it.

**Review by phargogh — APPROVED** [review]
Looks good to me!

**Pull request merged**
